packages/decentraland-ecs/src/decentraland/Input.ts

```diff
@@ -103,76 +127,122 @@ export class Input {
    *
    * Returns a function that can be called to remove the subscription.
    * @param eventName - The name of the event (see InputEventKind).
+   * @param pointerId - The id of the button.
+   * @param useRaycast - Enables getting raycast information.
    * @param fn - A callback function to be called when the event is triggered.
    */
-  public subscribe(eventName: InputEventKind, fn: (e: LocalPointerEvent) => void) {
-    this.subscriptions[eventName].push(fn)
-    return () => this.unsubscribe(eventName, fn)
+  public subscribe(
+    eventName: InputEventKind,
+    pointerId: ActionButton,
```

Comment:
Change for actionId or button to be consistent:
```suggestion
    button: ActionButton,
```